dotcom-rendering/src/web/layouts/LiveLayout.tsx

```diff
@@ -805,6 +805,7 @@ export const LiveLayout = ({ CAPIArticle, NAV, format }: Props) => {
 													filterKeyEvents={
 														CAPIArticle.filterKeyEvents
 													}
+													id="1"
```

Comment:
Could this `id` values be more descriptive ie `filter-key-events-toggle-desktop`?

Reply:
Great tip, thanks Joe!